screens/OnboardingScreen/Intro1.tsx

```diff
@@ -0,0 +1,47 @@
+import { Linking, StyleSheet } from "react-native";
+import { Text, View } from "../../components/Themed";
+import React from "react";
+import AIntroScreen from "./AIntroScreen";
+
+export default function Intro1() {
+  return (
+    <AIntroScreen headerText="Welcome to the Video Diary.">
+      {
+        "You will record a series of short videos describing your day. The next few pages explain how to do it. The diary should take approximately 20-30 minutes to complete."
+      }
+      {"\n"}
+      {"\n"}
+      {
+        "If you have questions or concerns, please contact a research staff member "
+      }
```

Comment:
Can these be combined into the same string?

Reply:
yes